### DISCUSSION TITLE
I'm still having noticable input lag when compared to stable.

### DISCUSSION THREAD

**Discussion by Paladynee** [ppy/osu#21911] [Q&A] [answered]
I dont know if its tied to the game audio latency being a bit different or the hitsounds playing a little late, i'm 99% sure i have more input lag compared to stable. Its not THAT bad but its still making me play worse. Im on fullscreen and im using unlimited framerate.

I think i have an idea why this happens. osu lazer "seems" to have some sort of antialiasing set in as far as i can see, that MIGHT be causing the input lag. i have no other ideas, if this does not get fixed, i might have to spend a week or 2 getting used to the input lag when the migration happens and stable no longer gets updated.

**Comment by MohaMG25** [comment]
I have a little bit of latency issue on my machine , im using a Gaming laptop with a decent properties and it can run stable with 480fps and 1.2ms latency. But when i play lazer i can feel a bit of delay happening infront of me and when i go back to play stable its just run smoothly.

**Comment by okolenmi** [comment]
I can confirm that there is small input lag from tablet (keyboard is ok as I can judje from 24ms original latency and ~24ms average error without offset enabled). Extra latency is approximately 5ms for me. Original latency of my tablet is 4ms, but in game it looks like ~10ms. I can see obvious latency difference between using tablet out of game and in game. I'm using ingame tablet driver. Average fps in game is 700-800 (1.1 ms latency displaying).

**Comment by peppy** [comment, marked as answer]
This means the game is not running exclusive fullscreen. Please wait for the upcoming renderer pipeline change which will fix this.
